### PR TITLE
[#1715] Align EOSC Marketplace footer with EOSC Portal

### LINKED ISSUE #1715: Align EOSC Marketplace footer with EOSC Portal footer
**Is your feature request related to a problem? Please describe.**
The Footer on the EOSC Marketplace is different from the footer of EOSC-Portal.eu.
There is a need to align the footer of the EOSC Marketplace so as to be equal to the one of the EOSC-Portal.eu.

**Describe the solution you'd like**
The current footer of EOSC Marketplace should be equal to the footer of EOSC Portal

**Additional context**
Current Marketplace footer:

EOSC Portal reference footer:

### PR DESCRIPTION
**Pull request by kmarszalek**
using https://www.eosc-portal.eu/ footer as a reference

Closes #1715

### CODE REVIEW

**Review by wujuu — COMMENTED** [review]
The content of the footer is the same as the one on EOSC Portal, but the looks is slightly off: font boldness, color, and we don't have this thing:
image

Idk if the looks should be exactly the same though.

**Comment by abacz** [comment]
Any css work should go to @jarekzet.

**Comment by jarekzet** [comment]
Added some CSS mods and up arrow to unify footer with EOSC portal. Also there are some bugs on EOSC portal footer (linking, columns header titles) and I made our footer clean of those.

**Review by wujuu — APPROVED** [review]

File: app/views/layouts/_footer.html.haml [hunk 1]
```diff
@@ -124,25 +118,41 @@
           %li.mb-1
             %a{ href: "#{portal_base}/media/past-events" }
               = _("Past Events")
+          %li.mb-1
+            %a{ href: "#{portal_base}/contribute-to-eosc" }
+              = _("Submit your News/Event")
+          %li.mb-1
+            = _("Documents")
```
Comment (JanKapala) [inline app/views/layouts/_footer.html.haml:125]: Shouldn't be a link here?

**Review by JanKapala — APPROVED** [review]

File: app/views/layouts/_footer.html.haml (hunk 1 above)
Comment (wujuu) [reply app/views/layouts/_footer.html.haml:125]: In the footer on the eosc.eu website, it's just a 'Documents' section, not a link to any particular resource

Comment (kmarszalek) [reply app/views/layouts/_footer.html.haml:125]: nope

**Pull request merged**